client/java/src/main/java/io/openlineage/client/utils/GravitinoUtils.java

```diff
@@ -0,0 +1,23 @@
+/*
+/* Copyright 2018-2025 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.client.utils;
+
+import io.openlineage.client.utils.gravitino.GravitinoInfoProviderImpl;
+import java.util.Arrays;
+
+public class GravitinoUtils {
+
+  private static final String DEFAULT_CATALOG_NAME = "spark_catalog";
```

Comment:
If this is the name of Spark config key `spark.sql.catalog.${catalog_name}`, then it just an arbitrary user-defined alias for Spark entity, not a name of Gravitino catalog. It should not be used as dataset name